src/utils/graphql/mutations/updateReview.mutation.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  mutation updateReview(
+    $reviewId: Int!
+    $trigger: Trigger = ON_REVIEW_SUBMIT
```

Comment:
Just checking -- is this query only used for submission? If if is, then maybe rename to `submitReview` so it's clear. 
And if not, then the Trigger will have to change accordingly.

Reply:
Sure. Good point. I was trying to use the same verb as the query/mutation. But it can be messy if we allow doing other changes that aren't meant to be for submission with this one.